modules/lang-painless/src/main/java/org/elasticsearch/painless/node/EAssignment.java

```diff
@@ -43,6 +43,7 @@ public AExpression getRightNode() {
         return rightNode;
     }
 
+    // TODO(stu): what does this mean?
```

Comment:
Don't forget to remove this to do. `postIfRead` is used to determine when a value is read from (either prior to a compound assignment or following a compound assignment). See comments on line 918 of DefaultUserTreeToIRTreePhase.java.

Reply:
Thanks, removed.